packages/web-pkg/README.md

```diff
@@ -2,8 +2,6 @@
 
 `web-pkg` is a package that provides utilities, most importantly a variety of components and composables, that can be useful when developing apps and extensions for ownCloud Web.
 
-It's important to note that this package solely provides types because `web-pkg` is intended solely for Web extension development. The actual implementation of the utilities is provided by the Web runtime automatically.
```

Comment:
Should we maybe add a sentence here, that `web-pkg` needs to be a `devDependency`? That's not a new information here, but still, would be good to have it.

Reply:
that would be great! 💪